Tasks/JenkinsQueueJob/job.ts

```diff
@@ -249,15 +249,15 @@ export class Job {
             var resultCode: string = this.parsedExecutionResult.result.toUpperCase();
             // codes map to fields in http://hudson-ci.org/javadoc/hudson/model/Result.html
             if (resultCode == 'SUCCESS') {
-                return 'Success';
+                return 'succeeded';
             } else if (resultCode == 'UNSTABLE') {
-                return 'Unstable';
+                return 'unstable';
             } else if (resultCode == 'FAILURE') {
-                return 'Failure';
+                return 'failed';
             } else if (resultCode == 'NOT_BUILT') {
-                return 'Not built';
+                return 'not built';
             } else if (resultCode == 'ABORTED') {
-                return 'Aborted';
+                return 'aborted';
```

Comment:
Not that we need to address it now, but I wonder if Jenkins could return non-English strings (instead of "Success").